tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+from tests import (  # noqa
```

Comment:
Hmm, are you certain you didn't change anything else in the same commit that caused it to find the tests? It should discover them all without this